### PR TITLE
Ft user should be able to register 154245374

### PR DESCRIPTION
**Pull request by owenbob**
#### What does this PR do?. 
- PR enables  user to register and post their details to the database
#### Description of Task to be completed?
-In order to complete this shall need to redirect to the login page.This shall be handled in the Login story.
#### What are the relevant pivotal tracker stories?
#154245374
#### Screenshots (if appropriate)
image

### CODE REVIEW

File: src/components/login.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+import './styling.css';
+
+const style = {
```
Comment (CeciliaCaroline) [inline src/components/login.js:6]: Always put your styles in a CSS file

File: src/App.js [hunk 2]
```diff
@@ -1,19 +1,37 @@
...
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import DefaultUI  from './components/materialUI';
+import Register from './components/register';
+import Login from './components/login';
+import Paper from 'material-ui/Paper';
+//import './mystyles.css';
+
 import './App.css';
 
+const style = {
```
Comment (CeciliaCaroline) [inline src/App.js:11]: Always put your styles in a CSS folder

File: src/components/register.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+import axios from 'axios';
+import './styling.css';
+
+
+const style = {
```
Comment (CeciliaCaroline) [inline src/components/register.js:8]: Do as earlier commented

File: src/components/register.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+          password:''
+        };
+        this.handleChange = this.handleChange.bind(this);
+      } 
+
+    handleClick = (e) =>{
+        e.preventDefault();
+        
+        // console.log("values",this.state.username,this.state.email,this.state.password);
+        console.log("Reguster \\")
```
Comment (CeciliaCaroline) [inline src/components/register.js]: What is the purpose of this log?

File: src/components/register.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+         const registerUrl='http://127.0.0.1:5000/register';
+        // let self = this;
```
Comment (CeciliaCaroline) [inline src/components/register.js:29]: Avoid leaving commented out code within the working code

File: src/components/register.js (hunk 4 above)
Comment (CeciliaCaroline) [inline src/components/register.js:27]: What is the purpose of this log?

File: src/components/login.js (hunk 1 above)
Comment (owenbob) [reply src/components/login.js:6]: Okay, this has been noted.Thanks for the feedback

File: src/App.js (hunk 2 above)
Comment (owenbob) [reply src/App.js:11]: This shall be improved in upcoming PRs.Thanks for the feedback

File: src/components/register.js (hunk 4 above)
Comment (owenbob) [reply src/components/register.js:27]: This was a debugging log.I shall eliminate these moving forward.

File: src/components/register.js (hunk 5 above)
Comment (owenbob) [reply src/components/register.js:29]: This has been noted ,Thanks

**Pull request merged**